src/main.ts

```diff
@@ -227,6 +235,26 @@ function toClosureJS(
       let {output} = tsickle.processES5(
           fileName, moduleId, jsFiles.get(fileName)!, cliSupport.pathToModuleName);
       jsFiles.set(fileName, output);
+    } else {
+      const tscSourceMapJson: any = jsFiles.get(fileName);
+      const tscSourceMap = new SourceMapConsumer(tscSourceMapJson);
+      const tscSourceMapGenerator = SourceMapGenerator.fromSourceMap(tscSourceMap);
+
+      for (const sourceFileName of (new SourceMapConsumer(tscSourceMapJson) as any).sources) {
```

Comment:
Again, the d.ts doesn't cover the fact that the SourceMapConsumer object has a sources field.